### DISCUSSION TITLE
Influx Authentification

### DISCUSSION THREAD

**Discussion by miurlit** [StefaE/PVForecast#5] [Q&A]
Hello,
thank you very much for sharing this nice tool. I have a question/issue - I do have authentification enabled in Influxdb and would like to use the Solcast (light) python script, but I failed, since I cannot write into the database and getting an authentication error.
I not very familiar with Python, but what I already tried is to add username and pwd to the config file and added these arguments into the "inlflux.py" file in the init section:
```
 def __init__(self, config):
         self.config     = config
         self._host      = self.config['Influx'].get('host', 'localhost')
         self._port      = self.config['Influx'].getint('port', 8086)
         self._database  = self.config['Influx'].get('database', None)
         self._token     = self.config['Influx'].get('token', None)
 	 self._username  = self.config['Influx'].get('username', None)
         self._password     = self.config['Influx'].get('password', None)
         self._org       = self.config['Influx'].get('org', None)
         self._influx_V2 = self.config['Influx'].getboolean('influx_v2', False)
```

This did not help -> same authentifcation error.
Any idea?
Thank you

Edit: I am using Influxdb 1.8

**Comment by StefaE** [comment]
I moved this over to [_Issues_](https://github.com/StefaE/PVForecast/issues/6)
